.github/workflows/prod-build.yml

```
@@ -0,0 +1,71 @@
+name: 'Production Release'
+
+on:
+  workflow_dispatch:
+    inputs:
+      package:
+        required: true
+        type: choice
+        description: Which package should be published?
+        options:
+          - vue
+          - angular
+          - react
+      version:
+        required: true
+        type: choice
+        description: Which version should be published?
+        options:
+          - patch
+          - minor
+          - major
+          - prepatch
+          - preminor
+          - premajor
+          - prerelease
+      tag:
+        required: true
+        type: choice
+        description: Which npm tag should this be published to?
+        options:
+          - latest
+          - next
+      preid:
+        type: choice
+        description: Which prerelease identifier should be used? This is only needed when version is "prepatch", "preminor", "premajor", or "prerelease".
+        options:
+          - ''
+          - alpha
+          - beta
+          - rc
+          - next
+
+jobs:
+  dev-build:
```

Should this be 'prod'?
```suggestion
  prod-build:
```

---

Done. Didn't take the suggestion because there was one other place.